components/ClaimSuccessPage/ClaimSuccessPage.tsx

```diff
@@ -0,0 +1,202 @@
+                      >
+                        <Image
+                          src={
+                            themeMode === "light"
+                              ? "/assets/Claim/Success/twitter.svg"
+                              : "/assets/Claim/Success/dark_twitter.svg"
+                          }
+                          width={21}
+                          height={17}
+                          alt="not found image"
+                        />
+                        Follow For Updates
+                      </Button>
+                      <Button
+                        id="view_passport_btn"
+                        className="mt-[20px] md:mt-[40px] md:mt-0 py-0 h-[49px] w-[291px] md:w-[284px] !px-0"
+                      >
+                        <Image
+                          src={
+                            themeMode === "light"
+                              ? "/assets/Claim/Success/opensea.svg"
+                              : "/assets/Claim/Success/dark_opensea.svg"
+                          }
+                          width={themeMode === "light" ? 33 : 34}
+                          height={themeMode === "light" ? 33 : 34}
+                          alt="not found  image"
+                        />
+                        View Passport
+                      </Button>
+                      <Button
+                        id="follow_for_btn"
+                        className="mt-[20px] md:mt-[40px] md:mt-0 py-0 h-[49px] w-[291px] !px-0 md:hidden"
+                      >
+                        <Image
+                          src={
+                            themeMode === "light"
+                              ? "/assets/Claim/Success/twitter.svg"
+                              : "/assets/Claim/Success/dark_twitter.svg"
+                          }
+                          width={21}
+                          height={17}
+                          alt="not found image"
+                        />
+                        Follow For Updates
+                      </Button>
+                    </div>
+                  </div>
+                </div>
+                <div className="justify-center md:flex hidden md:translate-y-[-5px]">
+                  {width && (
+                    <Media
+                      type="image"
+                      link="/assets/Claim/Success/pass.svg"
+                      containerClasses="rounded-[10px] overflow-hidden z-[1]"
+                      containerStyle={{
+                        width: isResponsive ? `${(width / 1440) * 465}px` : "465px",
+                        height: isResponsive ? `${(width / 1440) * 464}px` : "464px",
+                      }}
+                    />
+                  )}
+                </div>
+              </div>
+            </div>
+            <Footer className="!pt-0 !bg-transparent" />
+          </div>
+        )}
+      </div>
+    </Layout>
+  )
+}
```

Comment:
kinda long

any way you can break this into smaller, more modular, components?